diff --git a/node_modules/expo/Expo.podspec b/node_modules/expo/Expo.podspec
index 0c2b8a7..758c3ab 100644
--- a/node_modules/expo/Expo.podspec
+++ b/node_modules/expo/Expo.podspec
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.license        = package['license']
   s.author         = package['author']
   s.homepage       = package['homepage']
-  s.platform       = :ios, '12.0'
+  s.platforms      = { :ios => '12.0', :tvos => '12.0' }
   s.source         = { git: 'https://github.com/expo/expo.git' }
   s.static_framework = true
   s.header_dir     = 'Expo'
