diff --git a/node_modules/expo-keep-awake/ios/ExpoKeepAwake.podspec b/node_modules/expo-keep-awake/ios/ExpoKeepAwake.podspec
index 7a245ae..12c8121 100644
--- a/node_modules/expo-keep-awake/ios/ExpoKeepAwake.podspec
+++ b/node_modules/expo-keep-awake/ios/ExpoKeepAwake.podspec
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.license        = package['license']
   s.author         = package['author']
   s.homepage       = package['homepage']
-  s.platform       = :ios, '13.0'
+  s.platforms       = { :ios => '13.0', :tvos => '13.0'}
   s.swift_version  = '5.4'
   s.source         = { git: 'https://github.com/expo/expo.git' }
   s.static_framework = true
