diff --git a/node_modules/expo-splash-screen/ios/EXSplashScreen.podspec b/node_modules/expo-splash-screen/ios/EXSplashScreen.podspec
index 206daa0..81ca98c 100644
--- a/node_modules/expo-splash-screen/ios/EXSplashScreen.podspec
+++ b/node_modules/expo-splash-screen/ios/EXSplashScreen.podspec
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.license        = package['license']
   s.author         = package['author']
   s.homepage       = package['homepage']
-  s.platform       = :ios, '13.0'
+  s.platforms      = { :ios => '13.0', :tvos => '13.0' }
   s.source         = { git: 'https://github.com/expo/expo.git' }
   s.static_framework = true
 
