diff --git a/node_modules/expo-eas-client/ios/EASClient.podspec b/node_modules/expo-eas-client/ios/EASClient.podspec
index 7534d38..2b41f35 100644
--- a/node_modules/expo-eas-client/ios/EASClient.podspec
+++ b/node_modules/expo-eas-client/ios/EASClient.podspec
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.license        = package['license']
   s.author         = package['author']
   s.homepage       = package['homepage']
-  s.platform       = :ios, '12.0'
+  s.platforms      = { :ios => '12.0', :tvos => '12.0' }
   s.swift_version  = '5.4'
   s.source         = { git: '' }
   s.static_framework = true
