diff --git a/node_modules/expo-file-system/ios/EXFileSystem.podspec b/node_modules/expo-file-system/ios/EXFileSystem.podspec
index c1ccb12..fea59a6 100644
--- a/node_modules/expo-file-system/ios/EXFileSystem.podspec
+++ b/node_modules/expo-file-system/ios/EXFileSystem.podspec
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.license        = package['license']
   s.author         = package['author']
   s.homepage       = package['homepage']
-  s.platform       = :ios, '13.0'
+  s.platforms       = { :ios => '13.0', :tvos => '13.0'}
   s.swift_version  = '5.4'
   s.source         = { :git => 'https://github.com/expo/expo.git' }
   s.static_framework = true
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXFileSystem.m b/node_modules/expo-file-system/ios/EXFileSystem/EXFileSystem.m
index 1877d38..5f733ec 100644
--- a/node_modules/expo-file-system/ios/EXFileSystem/EXFileSystem.m
+++ b/node_modules/expo-file-system/ios/EXFileSystem/EXFileSystem.m
@@ -1069,10 +1069,16 @@ EX_EXPORT_METHOD_AS(getTotalDiskCapacityAsync, getTotalDiskCapacityAsyncWithReso
 // '<ARCType> *__autoreleasing*' problem solution: https://stackoverflow.com/a/8862061/4337317
 - (NSNumber *)freeDiskStorageWithError:(out NSError * __autoreleasing *)error
 {
-  NSDictionary *results = [self documentFileResourcesForKeys:@[NSURLVolumeAvailableCapacityForImportantUsageKey]
+#if TARGET_OS_TV
+  NSURLResourceKey key = NSURLVolumeAvailableCapacityKey;
+#else
+  NSURLResourceKey = NSURLVolumeAvailableCapacityForImportantUsageKey;
+#endif
+
+  NSDictionary *results = [self documentFileResourcesForKeys:@[key]
                                                        error:error];
 
-  return results[NSURLVolumeAvailableCapacityForImportantUsageKey];
+  return results[key];
 }
 
 @end
